cmd/cstor-volume-grpc/api/handler.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/openebs/maya/pkg/util"
+	"golang.org/x/net/context"
+)
+
+// sudo $ISTGTCONTROL snapdestroy vol1 snapname1 0
+// sudo $ISTGTCONTROL snapcreate vol1 snapname1
+
+// constants
+const (
+	VolumeGrpcListenPort = 7777
+	CmdSnapCreate        = "SNAPCREATE"
+	CmdSnapDestroy       = "SNAPDESTROY"
+	IoWaitTime           = 10
+	TotalWaitTime        = 60
+)
+
+//ApiUnixSockVar is unix socker variable
+var ApiUnixSockVar util.UnixSock
+
+// Server represents the gRPC server
+type Server struct {
+}
+
+// RunVolumeCommand generates response to a RunCommand request
+func (s *Server) RunVolumeCommand(ctx context.Context, in *VolumeCommand) (*VolumeCommand, error) {
+	log.Printf("Received command %s", in.Command)
```

Comment:
Using consistent logging library would be a good idea, I see that glog has also been used.

Reply:
+1